### ISSUE TITLE
More features for HTML version of Open Canvas

### ISSUE THREAD

**Issue by shashikanth171** [deimidis/privacy-journalists#1]
I love that you made a HTML version of Open Canvas. It would be great to have a separate repository for Open Canvas.
Can the following features added.
1. A Pop-up or a separate page for each part which can be accessed by click on the particular box
2. Edit info on canvas by editing easily readable files(something like yaml format)

**Comment by deimidis** [comment]
Thanks @shashikanth171 I wil try to make 1 soon, it's a great idea. I will need to investigate and learn more about the 2 item, indeed is a good idea. I also imagined to make an html file from different markdown files.

Progression over time that another team member is doing is a great idea too

**Comment by shashikanth171** [comment]
Yes. Item 2 can reduce time needed to edit a Canvas by many folds. I really appreciate it. Thanks!

**Comment by deimidis** [comment]
@shashikanth171 what other information you will need to add on specific pages for each section on the canvas model?

**Comment by rushRatnayake** [comment]
Hi, I would like to work on this in English. Would you mind if I work on it? Thanks :)
